recipes/leftbar/general_row/leftbar_general_row_icon.vue

```diff
@@ -45,59 +44,41 @@ export default {
   },
 
   computed: {
-    shouldRenderWithIcon () {
-      return this.type !== LEFTBAR_GENERAL_ROW_TYPES.DIALBOT && this.type !== LEFTBAR_GENERAL_ROW_TYPES.CONTACT_CENTER;
+    isIconType () {
+      return ![TYPES.DIALBOT, TYPES.CONTACT_CENTER].includes(this.type);
+    },
+
+    isContactCenterType () {
+      return this.type === TYPES.CONTACT_CENTER;
+    },
+
+    isDialbotType () {
+      return this.type === TYPES.DIALBOT;
+    },
+
+    getIconName () {
+      return ICON_MAPPING[this.type];
     },
 
     contactCenterIconClasses () {
       return [
-        'd-icon',
-        'leftbar-general-row__contact-center-icon',
-        LEFTBAR_GENERAL_ROW_CONTACT_CENTER_COLORS[this.color],
+        'd-bar2',
+        'dt-leftbar-row-icon-cc--size',
```

Comment:
`dt-leftbar-row-icon-cc--size` → `dt-leftbar-row__icon-cc`

BEM convention `__` is a _child_ of `dt-leftbar-row` (albeit not a direct child). No need to be specifc to `size` as it may include more properties.

In fact, you can remove `d-bar2` and add `border-radius: var(--size-200)` to it. 

Aside: ultimately with Design Tokens all radii would replace those, i.e. `border-radius: var(--dt-size-radius-200)`